cli/medperf/entities/interface.py

```diff
@@ -5,15 +5,14 @@
 class Entity(ABC):
     @abstractmethod
     def all(
-        cls, local_only: bool = False, comms_func: callable = None
+        cls, local_only: bool = False, filters: dict = None
```

Comment:
This fix was not connected to ls valid feature, just a bugfix: making abstract function signature relevant to a real implementations